nautobot/dcim/tables/locations.py

```diff
@@ -72,8 +72,18 @@ class Meta(BaseTable.Meta):
             "parent",
             "tenant",
             "description",
-            "tags",
             "actions",
+            "facility",
+            "asn",
+            "time_zone",
+            "physical_address",
+            "shipping_address",
```

Comment:
Feels weird to me to let these be table columns, but if it's feature parity with what SiteTable provides now, OK.